Library/Homebrew/style.rb

```diff
@@ -164,17 +164,24 @@ def run_rubocop(files, output_type,
       end
     end
 
-    def run_shellcheck(files, output_type)
+    def run_shellcheck(files, output_type, fix: false)
       files = shell_scripts if files.blank?
 
-      args = ["--shell=bash", "--enable=all", "--external-sources", "--source-path=#{HOMEBREW_LIBRARY}", "--", *files]
+      files = files.map(&:realpath)
+
+      args = ["--shell=bash", "--enable=all", "--external-sources", "--source-path=#{HOMEBREW_LIBRARY}", "--"]
```

Comment:
Should I add `"--exclude=SC2310,SC2311,SC2312"` option here or add `# shellcheck=...` in individual file? There are too many files involved.

https://github.com/koalaman/shellcheck/wiki/SC2310
https://github.com/koalaman/shellcheck/wiki/SC2311
https://github.com/koalaman/shellcheck/wiki/SC2312

Reply:
As a temporary fix for CI (I need to get other PRs unblocked), I've applied this exclusion in #12393.